docs/api-reference/operator.istio.io.md

```diff
@@ -0,0 +1,3242 @@
+# API Reference
+
+## Packages
+- [operator.istio.io/v1alpha1](#operatoristioiov1alpha1)
+
+
+## operator.istio.io/v1alpha1
+
+Package v1alpha1 contains API Schema definitions for the operator.istio.io v1alpha1 API group
+
+### Resource Types
+- [Istio](#istio)
+- [IstioCNI](#istiocni)
+- [IstioCNIList](#istiocnilist)
+- [IstioList](#istiolist)
+- [IstioRevision](#istiorevision)
+- [IstioRevisionList](#istiorevisionlist)
+
+
+
+#### ArchConfig
```

Comment:
@luksa This configuration came from the `values_type_gen.go` file.  And for some reason is being shown first. There is some ways to filter which fields we want to show, should be doing some filtering here?

Reply:
The objects are listed in alphabetical order, that's why ArchConfig is first.